tests/tests-framework/src/main/java/org/eclipse/dirigible/tests/IDE.java

```diff
@@ -149,7 +161,8 @@ public void createNewBlankProject(String projectName) {
 
         workbench.createNewProject(projectName);
 
-        assertCreatedProject(projectName);
+        // assertCreatedProject(projectName);
```

Comment:
Where as the main logic might benefit from having a comment of the previous state, the tests should hold on to only the current state? Maybe we can delete the comments.